tworaven_apps/eventdata_queries/testing/datavers_file_upload_test.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+import json
+import requests  # http://docs.python-requests.org/en/master/
+
+dataverse_server = 'https://dataverse.harvard.edu'  # no trailing slash
+api_key = '623d7768-fa3d-42c5-8a03-16d473dc8953'    # generated from kripanshu's account
```

Comment:
Move: dataverse_server, api_key, and persistentId to settings

In base.py, add:

```
#  For depositing Dataverse data
#
DATAVERSE_SERVER = os.environ.get('DATAVERSE_SERVER', None)
DATAVERSE_API_KEY = os.environ.get('DATAVERSE_API_KEY', None)
DATASET_PERSISTENT_ID = os.environ.get('DATASET_PERSISTENT_ID', None)
```

In this file, use the variables from the settings...